release-notes/release-notes.json

```diff
@@ -1,4 +1,39 @@
 [
+    {
+        "version": "4.6.0",
+        "title": "Fiddler Everywhere v4.6.0",
+        "date": "August 31, 2023",
+        "items": [
+            {
+                "type": "New",
+                "entries": [
+                    {
+                        "title": "Show upstream proxy information in the connection popup at the lower right of the application window"
```

Comment:
"in the status-bar"? Lower-right of the application window sounds too vague to me.